modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -0,0 +1,86 @@
+import sdpTransform from 'sdp-transform';
+
+/**
+ * RTCSessionDescription implementation.
+ */
+export default class RTCSessionDescription {
+    /**
+     * RTCSessionDescription constructor.
+     * @param {object} [descriptionInitDict]
+     * @param {string} [descriptionInitDict.type] "offer" / "answer".
+     * @param {string|object} [descriptionInitDict.sdp] SDP string or
```

Comment:
> You should never load/use the ortc/RTCSessionDescription but, instead, pass a ,RTCSessionDescriptionInit (as per spec) which is just a common JS object with type and sdp fields.

Maybe if we would migrate the code to use `RTCSessionDescriptionInit` it would look better, but that is not the case right now. Still all WebRTC browsers support this deprecated approach, so I don't see why it would be ok to allow confusion with 'sdp' types in the constructor.